### ISSUE TITLE
Amplify transfer service is running ~ with ^0.6.2 / last version (^0.63)

### ISSUE THREAD

**Issue by Sultandi** [aws-amplify/amplify-flutter#1916]
### Description

Hi, after i updated to last version, i was getting notification "Amplify transfer service is running"  which was annoying with the app.
(I can confirm no transfer process is running too).
after trying several previous versions, this notification always appear with version > ^0.6.2 / last version (^0.63).

_I have read this thread issue too (on android platform), when got this issue on version ^0.6.2
https://github.com/aws-amplify/amplify-android/issues/1803_
and some suggest with:
https://github.com/aws-amplify/amplify-android/issues/1803#issuecomment-1169162080

after waiting for the latest version update which is currently version ^0.6.3, i still get the same issue

so with version ^0.6.2 / last version (^0.63), how solve this annoying notification ("transfer service is running") on flutter?

### Categories

- [ ] Analytics
- [ ] API (REST)
- [ ] API (GraphQL)
- [ ] Auth
- [ ] Authenticator
- [x] #1926
- [X] Storage

### Steps to Reproduce

1. Go to project
2. Run Main.dart

### Screenshots

![IMG_20220722_134130](https://user-images.githubusercontent.com/102506348/180385149-273a8556-1b0b-463a-8713-f52c35762d6a.jpg)

### Platforms

- [ ] iOS
- [X] Android

### Android Device/Emulator API Level

API 32+

### Environment

```bash
[✓] Flutter (Channel stable, 3.0.4, on Ubuntu 22.04 LTS 5.15.0-40-generic, locale en_US.UTF-8)
    • Flutter version 3.0.4 at /home/PATH/DEVELOPMENT/SDK/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 85684f9300 (3 weeks ago), 2022-06-30 13:22:47 -0700
    • Engine revision 6ba2af10bb
    • Dart version 2.17.5
    • DevTools version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
    • Android SDK at /home/PATH/Android/Sdk
    • Platform android-33, build-tools 33.0.0
    • ANDROID_HOME = /home/PATH/Android/Sdk
    • Java binary at: /opt/_jetbrains/toolbox/apps/AndroidStudio/ch-0/212.5712.43.2112.8609683/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at google-chrome

[✓] Linux toolchain - develop for Linux desktop
    • Ubuntu clang version 14.0.0-1ubuntu1
    • cmake version 3.22.1
    • ninja version 1.10.1
    • pkg-config version 0.29.2

[✓] Android Studio (version 4.0)
    • Android Studio at /opt/android-studio
    • Flutter plugin version 51.0.1
    • Dart plugin version 193.7547
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] Android Studio (version 2021.2)
    • Android Studio at /opt/_jetbrains/toolbox/apps/AndroidStudio/ch-0/212.5712.43.2112.8609683
    • Flutter plugin version 67.1.2
    • Dart plugin version 212.5744
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)

[✓] IntelliJ IDEA Community Edition (version 2022.1)
    • IntelliJ at /opt/_jetbrains/toolbox/apps/IDEA-C/ch-0/221.5921.22
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] VS Code (version 1.68.1)
    • VS Code at /usr/share/code
    • Flutter extension version 3.42.0

[✓] Connected device (3 available)
    • Redmi 5 Plus (mobile) • 74db68980904 • android-arm64  • Android 10 (API 29)
    • Linux (desktop)       • linux        • linux-x64      • Ubuntu 22.04 LTS 5.15.0-40-generic
    • Chrome (web)          • chrome       • web-javascript • Google Chrome 103.0.5060.114

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
Process finished with exit code 0
```

### Dependencies

```bash
Dart SDK 2.17.5
Flutter SDK 3.0.4
tradiesmate 1.0.0+1

dependencies:
- after_layout 1.2.0 [flutter]
- amplify_api 0.6.1 [amplify_api_android amplify_api_ios amplify_core amplify_flutter aws_common collection flutter meta plugin_platform_interface]
- amplify_auth_cognito 0.6.1 [amplify_auth_cognito_android amplify_auth_cognito_ios amplify_core aws_common collection flutter meta plugin_platform_interface]
- amplify_core 0.6.1 [aws_common collection flutter intl json_annotation meta plugin_platform_interface uuid]
- amplify_flutter 0.6.1 [amplify_core amplify_datastore_plugin_interface amplify_flutter_android amplify_flutter_ios aws_common collection flutter meta plugin_platform_interface]
- amplify_storage_s3 0.6.1 [amplify_storage_s3_android amplify_storage_s3_ios amplify_core aws_common flutter meta plugin_platform_interface]
- android_intent_plus 3.1.1 [flutter platform meta]
- animated_text_kit 4.2.2 [flutter characters]
- auto_route 4.0.1 [flutter path collection meta universal_html]
- back_button_interceptor 6.0.1 [collection flutter]
- badges 2.0.3 [flutter]
- bloc_concurrency 0.2.0 [bloc stream_transform]
- bot_toast 4.0.3 [flutter]
- camera 0.10.0 [camera_android camera_avfoundation camera_platform_interface camera_web flutter flutter_plugin_android_lifecycle quiver]
- connectivity_plus 2.3.6 [flutter connectivity_plus_platform_interface connectivity_plus_linux connectivity_plus_macos connectivity_plus_web connectivity_plus_windows]
- cupertino_icons 1.0.5
- device_info_plus 3.2.4 [flutter device_info_plus_platform_interface device_info_plus_macos device_info_plus_linux device_info_plus_web device_info_plus_windows]
- devicelocale 0.5.4 [flutter flutter_web_plugins]
- enum_to_string 2.0.1
- equatable 2.0.3 [collection meta]
- extended_nested_scroll_view 5.1.3 [flutter visibility_detector]
- fcm_config 3.3.7 [flutter firebase_core firebase_messaging flutter_local_notifications http path_provider]
- fcm_localization 1.0.1 [flutter fcm_config shared_preferences]
- firebase_dynamic_links 4.3.3 [firebase_core firebase_core_platform_interface firebase_dynamic_links_platform_interface flutter meta plugin_platform_interface]
- flutter 0.0.0 [characters collection material_color_utilities meta vector_math sky_engine]
- flutter_bloc 8.0.1 [flutter bloc provider]
- flutter_datetime_picker 1.5.1 [flutter]
- flutter_inappwebview 5.4.3+7 [flutter]
- flutter_keyboard_visibility 5.3.0 [meta flutter_keyboard_visibility_platform_interface flutter_keyboard_visibility_web flutter]
- flutter_keyboard_visibility_platform_interface 2.0.0 [flutter meta plugin_platform_interface]
- flutter_launcher_icons 0.9.3 [args image path yaml]
- flutter_localizations 0.0.0 [flutter intl characters clock collection material_color_utilities meta path vector_math]
- flutter_map 2.0.0 [async collection flutter http latlong2 meta polylabel positioned_tap_detector_2 proj4dart tuple vector_math]
- flutter_polyline_points 1.0.0 [flutter http]
- flutter_secure_storage 5.0.2 [meta flutter flutter_secure_storage_linux flutter_secure_storage_macos flutter_secure_storage_platform_interface flutter_secure_storage_web flutter_secure_storage_windows]
- flutter_signature_pad 3.0.1 [flutter]
- flutter_snake_navigationbar 0.6.1 [flutter]
- flutter_svg_provider 1.0.3 [flutter http flutter_svg]
- font_awesome_flutter 10.1.0 [flutter]
- formz 0.4.1
- fzregex 2.0.0 [flutter]
- geocoding 2.0.4 [flutter geocoding_platform_interface]
- geolocator 9.0.1 [flutter geolocator_platform_interface geolocator_android geolocator_apple geolocator_web geolocator_windows]
- google_fonts 3.0.1 [flutter http path_provider crypto]
- google_maps_flutter 2.1.8 [flutter flutter_plugin_android_lifecycle google_maps_flutter_platform_interface]
- intl 0.17.0 [clock path]
- latlong2 0.8.1 [intl]
- modal_bottom_sheet 2.1.0 [flutter]
- nimu_shimmer 0.0.1 [flutter]
- package_info_plus 1.4.2 [flutter package_info_plus_platform_interface package_info_plus_linux package_info_plus_macos package_info_plus_windows package_info_plus_web]
- path 1.8.1
- path_provider 2.0.11 [flutter path_provider_android path_provider_ios path_provider_linux path_provider_macos path_provider_platform_interface path_provider_windows]
- pdf 3.8.1 [archive barcode crypto image meta path_parsing vector_math xml]
- permission_handler 10.0.0 [flutter meta permission_handler_android permission_handler_apple permission_handler_windows permission_handler_platform_interface]
- permission_handler_platform_interface 3.7.0 [flutter meta plugin_platform_interface]
- photo_view 0.14.0 [flutter]
- printing 5.9.1 [ffi flutter flutter_web_plugins http image js meta pdf plugin_platform_interface]
- pull_to_refresh 2.0.0 [flutter]
- sizer 2.0.15 [universal_io flutter]
- sqflite 2.0.3 [flutter sqflite_common path]
- status_alert 1.0.0 [flutter flare_flutter]
- table_calendar 3.0.6 [flutter intl simple_gesture_detector]
- timeline_tile 2.0.0 [flutter]
- transparent_image 2.0.0
- tuple 2.0.0 [quiver]
- url_launcher 6.1.5 [flutter url_launcher_android url_launcher_ios url_launcher_linux url_launcher_macos url_launcher_platform_interface url_launcher_web url_launcher_windows]

transitive dependencies:
- amplify_api_android 0.6.1 [flutter]
- amplify_api_ios 0.6.1 [amplify_core flutter]
- amplify_auth_cognito_android 0.6.1 [flutter]
- amplify_auth_cognito_ios 0.6.1 [amplify_core flutter]
- amplify_datastore_plugin_interface 0.6.1 [amplify_core collection flutter meta]
- amplify_flutter_android 0.6.1 [flutter]
- amplify_flutter_ios 0.6.1 [amplify_core flutter]
- amplify_storage_s3_android 0.6.1 [flutter]
- amplify_storage_s3_ios 0.6.1 [flutter]
- archive 3.3.0 [crypto path]
- args 2.3.1
- async 2.8.2 [collection meta]
- aws_common 0.1.1 [async collection http meta stream_transform uuid]
- barcode 2.2.1 [meta qr]
- bloc 8.0.3 [meta]
- boolean_selector 2.1.0 [source_span string_scanner]
- camera_android 0.10.0 [camera_platform_interface flutter flutter_plugin_android_lifecycle stream_transform]
- camera_avfoundation 0.9.7+1 [camera_platform_interface flutter]
- camera_platform_interface 2.2.0 [cross_file flutter plugin_platform_interface stream_transform]
- camera_web 0.3.0 [camera_platform_interface flutter flutter_web_plugins stream_transform]
- characters 1.2.0
- charcode 1.3.1
- clock 1.1.0
- collection 1.16.0
- connectivity_plus_linux 1.3.1 [flutter connectivity_plus_platform_interface meta nm]
- connectivity_plus_macos 1.2.4 [connectivity_plus_platform_interface flutter]
- connectivity_plus_platform_interface 1.2.1 [flutter meta plugin_platform_interface]
- connectivity_plus_web 1.2.3 [connectivity_plus_platform_interface flutter_web_plugins flutter]
- connectivity_plus_windows 1.2.2 [connectivity_plus_platform_interface flutter]
- cross_file 0.3.3+1 [js meta]
- crypto 3.0.2 [typed_data]
- csslib 0.17.2 [source_span]
- dbus 0.7.4 [args ffi meta xml]
- device_info_plus_linux 2.1.1 [device_info_plus_platform_interface file flutter meta]
- device_info_plus_macos 2.2.3 [device_info_plus_platform_interface flutter]
- device_info_plus_platform_interface 2.3.0+1 [flutter meta plugin_platform_interface]
- device_info_plus_web 2.1.0 [device_info_plus_platform_interface flutter_web_plugins flutter]
- device_info_plus_windows 2.1.1 [device_info_plus_platform_interface ffi flutter win32]
- fake_async 1.3.0 [clock collection]
- ffi 1.2.1
- file 6.1.2 [meta path]
- firebase_core 1.19.1 [firebase_core_platform_interface firebase_core_web flutter meta]
- firebase_core_platform_interface 4.5.0 [collection flutter flutter_test meta plugin_platform_interface]
- firebase_core_web 1.6.6 [firebase_core_platform_interface flutter flutter_web_plugins js meta]
- firebase_dynamic_links_platform_interface 0.2.3+8 [firebase_core flutter meta plugin_platform_interface]
- firebase_messaging 11.4.4 [firebase_core firebase_core_platform_interface firebase_messaging_platform_interface firebase_messaging_web flutter meta]
- firebase_messaging_platform_interface 3.5.4 [firebase_core flutter meta plugin_platform_interface]
- firebase_messaging_web 2.4.4 [firebase_core firebase_core_web firebase_messaging_platform_interface flutter flutter_web_plugins js meta]
- flare_flutter 3.0.2 [collection flutter meta]
- flutter_keyboard_visibility_web 2.0.0 [flutter_keyboard_visibility_platform_interface flutter_web_plugins flutter]
- flutter_local_notifications 9.6.1 [clock flutter flutter_local_notifications_linux flutter_local_notifications_platform_interface timezone]
- flutter_local_notifications_linux 0.5.0+1 [flutter flutter_local_notifications_platform_interface dbus path xdg_directories]
- flutter_local_notifications_platform_interface 5.0.0 [flutter plugin_platform_interface]
- flutter_plugin_android_lifecycle 2.0.6 [flutter]
- flutter_secure_storage_linux 1.1.0 [flutter flutter_secure_storage_platform_interface]
- flutter_secure_storage_macos 1.1.0 [flutter flutter_secure_storage_platform_interface]
- flutter_secure_storage_platform_interface 1.0.0 [flutter plugin_platform_interface]
- flutter_secure_storage_web 1.0.2 [flutter flutter_web_plugins flutter_secure_storage_platform_interface js]
- flutter_secure_storage_windows 1.1.2 [flutter flutter_secure_storage_platform_interface]
- flutter_svg 1.1.0 [flutter meta path_drawing vector_math xml]
- flutter_test 0.0.0 [flutter test_api path fake_async clock stack_trace vector_math async boolean_selector characters charcode collection matcher material_color_utilities meta source_span stream_channel string_scanner term_glyph]
- flutter_web_plugins 0.0.0 [flutter js characters collection material_color_utilities meta vector_math]
- geocoding_platform_interface 2.0.1 [flutter meta plugin_platform_interface]
- geolocator_android 4.0.0 [flutter geolocator_platform_interface]
- geolocator_apple 2.1.4 [flutter geolocator_platform_interface]
- geolocator_platform_interface 4.0.5 [flutter plugin_platform_interface vector_math meta]
- geolocator_web 2.1.5 [flutter flutter_web_plugins geolocator_platform_interface]
- geolocator_windows 0.1.1 [flutter geolocator_platform_interface]
- google_maps_flutter_platform_interface 2.2.0 [collection flutter plugin_platform_interface stream_transform]
- html 0.15.0 [csslib source_span]
- http 0.13.4 [async http_parser meta path]
- http_parser 4.0.1 [collection source_span string_scanner typed_data]
- image 3.2.0 [archive meta xml]
- js 0.6.4
- json_annotation 4.5.0 [meta]
- lists 1.0.1 [meta]
- matcher 0.12.11 [stack_trace]
- material_color_utilities 0.1.4
- meta 1.7.0
- mgrs_dart 2.0.0 [unicode]
- nested 1.0.0 [flutter]
- nm 0.5.0 [dbus]
- package_info_plus_linux 1.0.5 [package_info_plus_platform_interface flutter path]
- package_info_plus_macos 1.3.0 [flutter]
- package_info_plus_platform_interface 1.0.2 [flutter meta plugin_platform_interface]
- package_info_plus_web 1.0.5 [flutter flutter_web_plugins http meta package_info_plus_platform_interface]
- package_info_plus_windows 1.0.5 [package_info_plus_platform_interface ffi flutter win32]
- path_drawing 1.0.0 [vector_math meta path_parsing flutter]
- path_parsing 1.0.0 [vector_math meta]
- path_provider_android 2.0.14 [flutter path_provider_platform_interface]
- path_provider_ios 2.0.9 [flutter path_provider_platform_interface]
- path_provider_linux 2.1.7 [ffi flutter path path_provider_platform_interface xdg_directories]
- path_provider_macos 2.0.6 [flutter path_provider_platform_interface]
- path_provider_platform_interface 2.0.4 [flutter platform plugin_platform_interface]
- path_provider_windows 2.0.7 [ffi flutter path path_provider_platform_interface win32]
- permission_handler_android 10.0.0 [flutter permission_handler_platform_interface]
- permission_handler_apple 9.0.4 [flutter permission_handler_platform_interface]
- permission_handler_windows 0.1.0 [flutter permission_handler_platform_interface]
- petitparser 5.0.0 [meta]
- platform 3.1.0
- plugin_platform_interface 2.1.2 [meta]
- polylabel 1.0.1 [collection]
- positioned_tap_detector_2 1.0.4 [flutter]
- process 4.2.4 [file path platform]
- proj4dart 2.0.0 [mgrs_dart wkt_parser meta]
- provider 6.0.3 [collection flutter nested]
- qr 3.0.1 [meta]
- quiver 3.1.0 [matcher]
- shared_preferences 2.0.15 [flutter shared_preferences_android shared_preferences_ios shared_preferences_linux shared_preferences_macos shared_preferences_platform_interface shared_preferences_web shared_preferences_windows]
- shared_preferences_android 2.0.12 [flutter shared_preferences_platform_interface]
- shared_preferences_ios 2.1.1 [flutter shared_preferences_platform_interface]
- shared_preferences_linux 2.1.1 [file flutter path path_provider_linux path_provider_platform_interface shared_preferences_platform_interface]
- shared_preferences_macos 2.0.4 [flutter shared_preferences_platform_interface]
- shared_preferences_platform_interface 2.0.0 [flutter]
- shared_preferences_web 2.0.4 [flutter flutter_web_plugins shared_preferences_platform_interface]
- shared_preferences_windows 2.1.1 [file flutter path path_provider_platform_interface path_provider_windows shared_preferences_platform_interface]
- simple_gesture_detector 0.2.0 [flutter]
- sky_engine 0.0.99
- source_span 1.8.2 [collection path term_glyph]
- sqflite_common 2.2.1+1 [synchronized path meta]
- stack_trace 1.10.0 [path]
- stream_channel 2.1.0 [async]
- stream_transform 2.0.0
- string_scanner 1.1.0 [charcode source_span]
- synchronized 3.0.0+2
- term_glyph 1.2.0
- test_api 0.4.9 [async boolean_selector collection meta source_span stack_trace stream_channel string_scanner term_glyph matcher]
- timezone 0.8.0 [path]
- typed_data 1.3.1 [collection]
- unicode 0.3.1 [lists]
- universal_html 2.0.8 [async csslib charcode collection html meta source_span typed_data universal_io]
- universal_io 2.0.4 [collection crypto meta typed_data]
- url_launcher_android 6.0.17 [flutter url_launcher_platform_interface]
- url_launcher_ios 6.0.17 [flutter url_launcher_platform_interface]
- url_launcher_linux 3.0.1 [flutter url_launcher_platform_interface]
- url_launcher_macos 3.0.1 [flutter url_launcher_platform_interface]
- url_launcher_platform_interface 2.1.0 [flutter plugin_platform_interface]
- url_launcher_web 2.0.11 [flutter flutter_web_plugins url_launcher_platform_interface]
- url_launcher_windows 3.0.1 [flutter url_launcher_platform_interface]
- uuid 3.0.6 [crypto]
- vector_math 2.1.2
- visibility_detector 0.3.3 [flutter]
- win32 2.6.1 [ffi]
- wkt_parser 2.0.0
- xdg_directories 0.2.0+1 [meta path process]
- xml 6.1.0 [collection meta petitparser]
- yaml 3.1.1 [collection source_span string_scanner]
sultandi@sul-Vostro:~/DEVELOPMENT/REPO/tradies-mate$ flutter pub deps --no-dev --style=compact
Dart SDK 2.17.5
Flutter SDK 3.0.4
tradiesmate 1.0.0+1

dependencies:
- after_layout 1.2.0 [flutter]
- amplify_api 0.6.1 [amplify_api_android amplify_api_ios amplify_core amplify_flutter aws_common collection flutter meta plugin_platform_interface]
- amplify_auth_cognito 0.6.1 [amplify_auth_cognito_android amplify_auth_cognito_ios amplify_core aws_common collection flutter meta plugin_platform_interface]
- amplify_core 0.6.1 [aws_common collection flutter intl json_annotation meta plugin_platform_interface uuid]
- amplify_flutter 0.6.1 [amplify_core amplify_datastore_plugin_interface amplify_flutter_android amplify_flutter_ios aws_common collection flutter meta plugin_platform_interface]
- amplify_storage_s3 0.6.1 [amplify_storage_s3_android amplify_storage_s3_ios amplify_core aws_common flutter meta plugin_platform_interface]
- android_intent_plus 3.1.1 [flutter platform meta]
- animated_text_kit 4.2.2 [flutter characters]
- auto_route 4.0.1 [flutter path collection meta universal_html]
- back_button_interceptor 6.0.1 [collection flutter]
- badges 2.0.3 [flutter]
- bloc_concurrency 0.2.0 [bloc stream_transform]
- bot_toast 4.0.3 [flutter]
- camera 0.10.0 [camera_android camera_avfoundation camera_platform_interface camera_web flutter flutter_plugin_android_lifecycle quiver]
- connectivity_plus 2.3.6 [flutter connectivity_plus_platform_interface connectivity_plus_linux connectivity_plus_macos connectivity_plus_web connectivity_plus_windows]
- cupertino_icons 1.0.5
- device_info_plus 3.2.4 [flutter device_info_plus_platform_interface device_info_plus_macos device_info_plus_linux device_info_plus_web device_info_plus_windows]
- devicelocale 0.5.4 [flutter flutter_web_plugins]
- enum_to_string 2.0.1
- equatable 2.0.3 [collection meta]
- extended_nested_scroll_view 5.1.3 [flutter visibility_detector]
- fcm_config 3.3.7 [flutter firebase_core firebase_messaging flutter_local_notifications http path_provider]
- fcm_localization 1.0.1 [flutter fcm_config shared_preferences]
- firebase_dynamic_links 4.3.3 [firebase_core firebase_core_platform_interface firebase_dynamic_links_platform_interface flutter meta plugin_platform_interface]
- flutter 0.0.0 [characters collection material_color_utilities meta vector_math sky_engine]
- flutter_bloc 8.0.1 [flutter bloc provider]
- flutter_datetime_picker 1.5.1 [flutter]
- flutter_inappwebview 5.4.3+7 [flutter]
- flutter_keyboard_visibility 5.3.0 [meta flutter_keyboard_visibility_platform_interface flutter_keyboard_visibility_web flutter]
- flutter_keyboard_visibility_platform_interface 2.0.0 [flutter meta plugin_platform_interface]
- flutter_launcher_icons 0.9.3 [args image path yaml]
- flutter_localizations 0.0.0 [flutter intl characters clock collection material_color_utilities meta path vector_math]
- flutter_map 2.0.0 [async collection flutter http latlong2 meta polylabel positioned_tap_detector_2 proj4dart tuple vector_math]
- flutter_polyline_points 1.0.0 [flutter http]
- flutter_secure_storage 5.0.2 [meta flutter flutter_secure_storage_linux flutter_secure_storage_macos flutter_secure_storage_platform_interface flutter_secure_storage_web flutter_secure_storage_windows]
- flutter_signature_pad 3.0.1 [flutter]
- flutter_snake_navigationbar 0.6.1 [flutter]
- flutter_svg_provider 1.0.3 [flutter http flutter_svg]
- font_awesome_flutter 10.1.0 [flutter]
- formz 0.4.1
- fzregex 2.0.0 [flutter]
- geocoding 2.0.4 [flutter geocoding_platform_interface]
- geolocator 9.0.1 [flutter geolocator_platform_interface geolocator_android geolocator_apple geolocator_web geolocator_windows]
- google_fonts 3.0.1 [flutter http path_provider crypto]
- google_maps_flutter 2.1.8 [flutter flutter_plugin_android_lifecycle google_maps_flutter_platform_interface]
- intl 0.17.0 [clock path]
- latlong2 0.8.1 [intl]
- modal_bottom_sheet 2.1.0 [flutter]
- nimu_shimmer 0.0.1 [flutter]
- package_info_plus 1.4.2 [flutter package_info_plus_platform_interface package_info_plus_linux package_info_plus_macos package_info_plus_windows package_info_plus_web]
- path 1.8.1
- path_provider 2.0.11 [flutter path_provider_android path_provider_ios path_provider_linux path_provider_macos path_provider_platform_interface path_provider_windows]
- pdf 3.8.1 [archive barcode crypto image meta path_parsing vector_math xml]
- permission_handler 10.0.0 [flutter meta permission_handler_android permission_handler_apple permission_handler_windows permission_handler_platform_interface]
- permission_handler_platform_interface 3.7.0 [flutter meta plugin_platform_interface]
- photo_view 0.14.0 [flutter]
- printing 5.9.1 [ffi flutter flutter_web_plugins http image js meta pdf plugin_platform_interface]
- pull_to_refresh 2.0.0 [flutter]
- sizer 2.0.15 [universal_io flutter]
- sqflite 2.0.3 [flutter sqflite_common path]
- status_alert 1.0.0 [flutter flare_flutter]
- table_calendar 3.0.6 [flutter intl simple_gesture_detector]
- timeline_tile 2.0.0 [flutter]
- transparent_image 2.0.0
- tuple 2.0.0 [quiver]
- url_launcher 6.1.5 [flutter url_launcher_android url_launcher_ios url_launcher_linux url_launcher_macos url_launcher_platform_interface url_launcher_web url_launcher_windows]

transitive dependencies:
- amplify_api_android 0.6.1 [flutter]
- amplify_api_ios 0.6.1 [amplify_core flutter]
- amplify_auth_cognito_android 0.6.1 [flutter]
- amplify_auth_cognito_ios 0.6.1 [amplify_core flutter]
- amplify_datastore_plugin_interface 0.6.1 [amplify_core collection flutter meta]
- amplify_flutter_android 0.6.1 [flutter]
- amplify_flutter_ios 0.6.1 [amplify_core flutter]
- amplify_storage_s3_android 0.6.1 [flutter]
- amplify_storage_s3_ios 0.6.1 [flutter]
- archive 3.3.0 [crypto path]
- args 2.3.1
- async 2.8.2 [collection meta]
- aws_common 0.1.1 [async collection http meta stream_transform uuid]
- barcode 2.2.1 [meta qr]
- bloc 8.0.3 [meta]
- boolean_selector 2.1.0 [source_span string_scanner]
- camera_android 0.10.0 [camera_platform_interface flutter flutter_plugin_android_lifecycle stream_transform]
- camera_avfoundation 0.9.7+1 [camera_platform_interface flutter]
- camera_platform_interface 2.2.0 [cross_file flutter plugin_platform_interface stream_transform]
- camera_web 0.3.0 [camera_platform_interface flutter flutter_web_plugins stream_transform]
- characters 1.2.0
- charcode 1.3.1
- clock 1.1.0
- collection 1.16.0
- connectivity_plus_linux 1.3.1 [flutter connectivity_plus_platform_interface meta nm]
- connectivity_plus_macos 1.2.4 [connectivity_plus_platform_interface flutter]
- connectivity_plus_platform_interface 1.2.1 [flutter meta plugin_platform_interface]
- connectivity_plus_web 1.2.3 [connectivity_plus_platform_interface flutter_web_plugins flutter]
- connectivity_plus_windows 1.2.2 [connectivity_plus_platform_interface flutter]
- cross_file 0.3.3+1 [js meta]
- crypto 3.0.2 [typed_data]
- csslib 0.17.2 [source_span]
- dbus 0.7.4 [args ffi meta xml]
- device_info_plus_linux 2.1.1 [device_info_plus_platform_interface file flutter meta]
- device_info_plus_macos 2.2.3 [device_info_plus_platform_interface flutter]
- device_info_plus_platform_interface 2.3.0+1 [flutter meta plugin_platform_interface]
- device_info_plus_web 2.1.0 [device_info_plus_platform_interface flutter_web_plugins flutter]
- device_info_plus_windows 2.1.1 [device_info_plus_platform_interface ffi flutter win32]
- fake_async 1.3.0 [clock collection]
- ffi 1.2.1
- file 6.1.2 [meta path]
- firebase_core 1.19.1 [firebase_core_platform_interface firebase_core_web flutter meta]
- firebase_core_platform_interface 4.5.0 [collection flutter flutter_test meta plugin_platform_interface]
- firebase_core_web 1.6.6 [firebase_core_platform_interface flutter flutter_web_plugins js meta]
- firebase_dynamic_links_platform_interface 0.2.3+8 [firebase_core flutter meta plugin_platform_interface]
- firebase_messaging 11.4.4 [firebase_core firebase_core_platform_interface firebase_messaging_platform_interface firebase_messaging_web flutter meta]
- firebase_messaging_platform_interface 3.5.4 [firebase_core flutter meta plugin_platform_interface]
- firebase_messaging_web 2.4.4 [firebase_core firebase_core_web firebase_messaging_platform_interface flutter flutter_web_plugins js meta]
- flare_flutter 3.0.2 [collection flutter meta]
- flutter_keyboard_visibility_web 2.0.0 [flutter_keyboard_visibility_platform_interface flutter_web_plugins flutter]
- flutter_local_notifications 9.6.1 [clock flutter flutter_local_notifications_linux flutter_local_notifications_platform_interface timezone]
- flutter_local_notifications_linux 0.5.0+1 [flutter flutter_local_notifications_platform_interface dbus path xdg_directories]
- flutter_local_notifications_platform_interface 5.0.0 [flutter plugin_platform_interface]
- flutter_plugin_android_lifecycle 2.0.6 [flutter]
- flutter_secure_storage_linux 1.1.0 [flutter flutter_secure_storage_platform_interface]
- flutter_secure_storage_macos 1.1.0 [flutter flutter_secure_storage_platform_interface]
- flutter_secure_storage_platform_interface 1.0.0 [flutter plugin_platform_interface]
- flutter_secure_storage_web 1.0.2 [flutter flutter_web_plugins flutter_secure_storage_platform_interface js]
- flutter_secure_storage_windows 1.1.2 [flutter flutter_secure_storage_platform_interface]
- flutter_svg 1.1.0 [flutter meta path_drawing vector_math xml]
- flutter_test 0.0.0 [flutter test_api path fake_async clock stack_trace vector_math async boolean_selector characters charcode collection matcher material_color_utilities meta source_span stream_channel string_scanner term_glyph]
- flutter_web_plugins 0.0.0 [flutter js characters collection material_color_utilities meta vector_math]
- geocoding_platform_interface 2.0.1 [flutter meta plugin_platform_interface]
- geolocator_android 4.0.0 [flutter geolocator_platform_interface]
- geolocator_apple 2.1.4 [flutter geolocator_platform_interface]
- geolocator_platform_interface 4.0.5 [flutter plugin_platform_interface vector_math meta]
- geolocator_web 2.1.5 [flutter flutter_web_plugins geolocator_platform_interface]
- geolocator_windows 0.1.1 [flutter geolocator_platform_interface]
- google_maps_flutter_platform_interface 2.2.0 [collection flutter plugin_platform_interface stream_transform]
- html 0.15.0 [csslib source_span]
- http 0.13.4 [async http_parser meta path]
- http_parser 4.0.1 [collection source_span string_scanner typed_data]
- image 3.2.0 [archive meta xml]
- js 0.6.4
- json_annotation 4.5.0 [meta]
- lists 1.0.1 [meta]
- matcher 0.12.11 [stack_trace]
- material_color_utilities 0.1.4
- meta 1.7.0
- mgrs_dart 2.0.0 [unicode]
- nested 1.0.0 [flutter]
- nm 0.5.0 [dbus]
- package_info_plus_linux 1.0.5 [package_info_plus_platform_interface flutter path]
- package_info_plus_macos 1.3.0 [flutter]
- package_info_plus_platform_interface 1.0.2 [flutter meta plugin_platform_interface]
- package_info_plus_web 1.0.5 [flutter flutter_web_plugins http meta package_info_plus_platform_interface]
- package_info_plus_windows 1.0.5 [package_info_plus_platform_interface ffi flutter win32]
- path_drawing 1.0.0 [vector_math meta path_parsing flutter]
- path_parsing 1.0.0 [vector_math meta]
- path_provider_android 2.0.14 [flutter path_provider_platform_interface]
- path_provider_ios 2.0.9 [flutter path_provider_platform_interface]
- path_provider_linux 2.1.7 [ffi flutter path path_provider_platform_interface xdg_directories]
- path_provider_macos 2.0.6 [flutter path_provider_platform_interface]
- path_provider_platform_interface 2.0.4 [flutter platform plugin_platform_interface]
- path_provider_windows 2.0.7 [ffi flutter path path_provider_platform_interface win32]
- permission_handler_android 10.0.0 [flutter permission_handler_platform_interface]
- permission_handler_apple 9.0.4 [flutter permission_handler_platform_interface]
- permission_handler_windows 0.1.0 [flutter permission_handler_platform_interface]
- petitparser 5.0.0 [meta]
- platform 3.1.0
- plugin_platform_interface 2.1.2 [meta]
- polylabel 1.0.1 [collection]
- positioned_tap_detector_2 1.0.4 [flutter]
- process 4.2.4 [file path platform]
- proj4dart 2.0.0 [mgrs_dart wkt_parser meta]
- provider 6.0.3 [collection flutter nested]
- qr 3.0.1 [meta]
- quiver 3.1.0 [matcher]
- shared_preferences 2.0.15 [flutter shared_preferences_android shared_preferences_ios shared_preferences_linux shared_preferences_macos shared_preferences_platform_interface shared_preferences_web shared_preferences_windows]
- shared_preferences_android 2.0.12 [flutter shared_preferences_platform_interface]
- shared_preferences_ios 2.1.1 [flutter shared_preferences_platform_interface]
- shared_preferences_linux 2.1.1 [file flutter path path_provider_linux path_provider_platform_interface shared_preferences_platform_interface]
- shared_preferences_macos 2.0.4 [flutter shared_preferences_platform_interface]
- shared_preferences_platform_interface 2.0.0 [flutter]
- shared_preferences_web 2.0.4 [flutter flutter_web_plugins shared_preferences_platform_interface]
- shared_preferences_windows 2.1.1 [file flutter path path_provider_platform_interface path_provider_windows shared_preferences_platform_interface]
- simple_gesture_detector 0.2.0 [flutter]
- sky_engine 0.0.99
- source_span 1.8.2 [collection path term_glyph]
- sqflite_common 2.2.1+1 [synchronized path meta]
- stack_trace 1.10.0 [path]
- stream_channel 2.1.0 [async]
- stream_transform 2.0.0
- string_scanner 1.1.0 [charcode source_span]
- synchronized 3.0.0+2
- term_glyph 1.2.0
- test_api 0.4.9 [async boolean_selector collection meta source_span stack_trace stream_channel string_scanner term_glyph matcher]
- timezone 0.8.0 [path]
- typed_data 1.3.1 [collection]
- unicode 0.3.1 [lists]
- universal_html 2.0.8 [async csslib charcode collection html meta source_span typed_data universal_io]
- universal_io 2.0.4 [collection crypto meta typed_data]
- url_launcher_android 6.0.17 [flutter url_launcher_platform_interface]
- url_launcher_ios 6.0.17 [flutter url_launcher_platform_interface]
- url_launcher_linux 3.0.1 [flutter url_launcher_platform_interface]
- url_launcher_macos 3.0.1 [flutter url_launcher_platform_interface]
- url_launcher_platform_interface 2.1.0 [flutter plugin_platform_interface]
- url_launcher_web 2.0.11 [flutter flutter_web_plugins url_launcher_platform_interface]
- url_launcher_windows 3.0.1 [flutter url_launcher_platform_interface]
- uuid 3.0.6 [crypto]
- vector_math 2.1.2
- visibility_detector 0.3.3 [flutter]
- win32 2.6.1 [ffi]
- wkt_parser 2.0.0
- xdg_directories 0.2.0+1 [meta path process]
- xml 6.1.0 [collection meta petitparser]
- yaml 3.1.1 [collection source_span string_scanner]
```

### Device

N/A

### OS

N/A

### CLI Version

9.0.0

### Additional Context

_temporary my pubspec (downgrade version of amplify)_
**amplify_storage_s3: 0.6.1 / 0.6.0**

**Comment by ragingsquirrel3** [comment]
Thanks for reporting this @Sultandi . This appears to be an issue with the latest version of amplify-android and we are working with them on a fix. We will update here with progress.

**Comment by miguelflores1993** [comment]
> Thanks for reporting this @Sultandi . This appears to be an issue with the latest version of amplify-android and we are working with them on a fix. We will update here with progress.

Is there any news about this problem?

**Comment by dnys1** [comment]
Hi @miguelflores1993 - yes, we are just waiting on a release by the Android team with the fix. As soon as that's ready (expected today or tomorrow), we will push a new version of Amplify Flutter and update this ticket. Thank you for your patience and apologies for the delay!

**Comment by haverchuck** [comment]
@miguelflores1993 Please give amplify-flutter 0.6.4 a try. I was able to reproduce your issue, and it was resolved with this latest release.

**Comment by djsjr** [comment]
@haverchuck @dnys1 Just tried out 0.6.4. "File transfer in progress" notification appears for several seconds and then goes away - with a little android icon. However, my app crashes half the time a transfer is initiated.

Aside from the crashing, this is still in no way a fix. It's unacceptable to have such a notification go out to users like this - I cannot ship this, even if it didn't crash.

Here's what I get when it crashes:
``` 
E/AndroidRuntime(16506): android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{88ec9e4 u0 com.website.appname/com.amazonaws.mobileconnectors.s3.transferutility.TransferService}
```

**Comment by miguelflores1993** [comment]
> @haverchuck @dnys1 Just tried out 0.6.4. "File transfer in progress" notification appears for several seconds and then goes away - with a little android icon. However, my app crashes half the time a transfer is initiated.
> 
> Aside from the crashing, this is still in no way a fix. It's unacceptable to have such a notification go out to users like this - I cannot ship this, even if it didn't crash.
> 
> Here's what I get when it crashes:
> 
> ```
> E/AndroidRuntime(16506): android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{88ec9e4 u0 com.website.appname/com.amazonaws.mobileconnectors.s3.transferutility.TransferService}
> ```
> 
> I've downgraded to 0.6.1 until this is resolved or changed.

I have the same problem.
It is very ugly that in two realeases there are very ugly errors, check well before publishing something.
I just posted the error

https://github.com/aws-amplify/amplify-flutter/issues/1935

**Comment by dnys1** [comment]
@djsjr @miguelflores1993 thanks for testing the fix, and apologies it introduced a new issue. The Android team is working on a fix and will have it ready shortly. We will update this ticket and #1935 when it is released.

**Comment by dnys1** [comment]
@Sultandi @djsjr @miguelflores1993 v0.6.5 has been published with a fix for the crash introduced by 0.6.4 and with a lower notification priority which should produce a notification with no sound/vibration.

Please note that background uploads/downloads are still unsupported in Android 12+. This is due to restrictions by the OS on starting foreground services in the background.

Further, we realize the notification is still less than ideal. We will continue working to correct both of these issues. In the meantime, you should not experience any crashes on devices running Android <12, and only on Android 12 in the event of a background upload/download. Any other aberrant behavior should be reported in a new GitHub issue.

Thank you all for your patience resolving this issue.

**Comment by dnys1** [comment]
Slight correction to the above statement. We do not expect a crash in any situation, including Android 12+ devices. For these devices, the OS simply restricts what can happen and we would expect a non-fatal exception to be thrown.
